gst/nnstreamer/tensor_common_pipeline.c

```diff
@@ -164,7 +164,7 @@ gst_tensor_time_sync_get_current_time (GstCollectPads * collect,
           /* fall-through */
         case SYNC_SLOWEST:
         case SYNC_REFRESH:
-          if (*current_time < GST_BUFFER_PTS (buf))
+          if (*current_time <= GST_BUFFER_PTS (buf))
```

Comment:
Q. Is it related to this PR?

Reply:
Not directly related the purpose of the this PR. I will make a new PR for this. Thanks.